.NET/Microsoft.Recognizers.Text.DateTime/French/Extractors/FrenchDateTimePeriodExtractorConfiguration.cs

```diff
@@ -81,37 +79,34 @@ public FrenchDateTimePeriodExtractorConfiguration() : base(DateTimeOptions.None)
         public static readonly Regex DateUnitRegex =
             new Regex(DateTimeDefinitions.DateUnitRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);
 
-        Regex IDateTimePeriodExtractorConfiguration.DateUnitRegex => DateUnitRegex;
-
-        public Regex FollowedUnit => TimeFollowedUnit;
-
-        Regex IDateTimePeriodExtractorConfiguration.NumberCombinedWithUnit => TimeNumberCombinedWithUnit;
-
-        Regex IDateTimePeriodExtractorConfiguration.TimeUnitRegex => TimeTimeUnitRegex;
+        public static readonly Regex PrefixPeriodRegex =
+            new Regex(DateTimeDefinitions.PrefixPeriodRegex,
+                RegexOptions.IgnoreCase | RegexOptions.Singleline | RegexOptions.RightToLeft);
 
-        Regex IDateTimePeriodExtractorConfiguration.RelativeTimeUnitRegex => RelativeTimeUnitRegex;
-
-        Regex IDateTimePeriodExtractorConfiguration.RestOfDateTimeRegex => RestOfDateTimeRegex;
-
-        public Regex PastPrefixRegex => FrenchDatePeriodExtractorConfiguration.PastPrefixRegex; // Note: FR 'past' i.e 'dernier' is a suffix following after, however interface enforces 'prefix' nomenclature
+        public IEnumerable<Regex> SimpleCasesRegex => SimpleCases;
 
-        public Regex NextPrefixRegex => FrenchDatePeriodExtractorConfiguration.NextPrefixRegex; // Note: FR 'next' i.e 'prochain' is a suffix following after, i.e 'lundi prochain', however 'prefix' is enforced by interface
+        public Regex PrepositionRegex => FrenchTimePeriodExtractorConfiguration.PrepositionRegex;
 
-        public Regex FutureSuffixRegex => FrenchDatePeriodExtractorConfiguration.FutureSuffixRegex;
+        public Regex TillRegex => FrenchTimePeriodExtractorConfiguration.TillRegex;
 
-        public Regex WeekDayRegex => new Regex(DateTimeDefinitions.WeekDayRegex, RegexOptions.IgnoreCase | RegexOptions.Singleline);
+        public Regex TimeOfDayRegex => PeriodTimeOfDayRegex;
 
-        Regex IDateTimePeriodExtractorConfiguration.PeriodTimeOfDayWithDateRegex => PeriodTimeOfDayWithDateRegex;
+        public Regex SpecificTimeOfDayRegex => PeriodSpecificTimeOfDayRegex;
 
-        Regex IDateTimePeriodExtractorConfiguration.GeneralEndingRegex => GeneralEndingRegex;
+        public Regex FollowedUnit => TimeFollowedUnit;
 
-        Regex IDateTimePeriodExtractorConfiguration.MiddlePauseRegex => MiddlePauseRegex;
+        public Regex PastPrefixRegex =>
+            FrenchDatePeriodExtractorConfiguration
+                .PastPrefixRegex; // Note: FR 'past' i.e 'dernier' is a suffix following after, however interface enforces 'prefix' nomenclature
```

Comment:
Can you move the comments immediately before the regex? It will make reading them easier.